pkg/patterns/declarative/pkg/applier/golden_test.go

```diff
@@ -0,0 +1,155 @@
+package applier
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"net/http"
+	"path/filepath"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/kubebuilder-declarative-pattern/applylib/applyset"
+	"sigs.k8s.io/kubebuilder-declarative-pattern/mockkubeapiserver"
+	"sigs.k8s.io/kubebuilder-declarative-pattern/pkg/patterns/declarative/pkg/manifest"
+	"sigs.k8s.io/kubebuilder-declarative-pattern/pkg/restmapper"
+	"sigs.k8s.io/kubebuilder-declarative-pattern/pkg/test/httprecorder"
+	"sigs.k8s.io/kubebuilder-declarative-pattern/pkg/test/testharness"
+)
+
+func fakeParent() runtime.Object {
+	parent := &unstructured.Unstructured{}
+	parent.SetKind("ConfigMap")
+	parent.SetAPIVersion("v1")
+	parent.SetName("test")
+	parent.SetNamespace("default")
+	return parent
+}
+
+func runApplierGoldenTests(t *testing.T, testDir string, interceptHTTPServer bool, applier Applier) {
```

Comment:
All 3 appliers needs this method. So I move it from `applyset.go` to non-tagged file

Reply:
OK, but it might be easier just not to tag that file, if that's an option.